src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/WorkPackageSummary.tsx

```diff
@@ -59,7 +59,7 @@ const WorkPackageSummary: React.FC<WorkPackageSummaryProps> = ({ workPackage })
             {workPackage.name}
           </Link>
         </Box>
-        {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
+        <WorkPackageStageChip stage={workPackage.stage} />
```

Comment:
if it has no chip we dont want it to show up at all so the previous code was right (this way it'll show up as ag rey chip saying "none")

Reply:
are you sure? I think none would actually be better imo?

Reply:
Per slack just change the default case in the text wp stage function to say “No Stage”